packages/fether-react/src/utils/transaction.spec.js

```diff
@@ -39,13 +39,13 @@ describe('estimateGas', () => {
   });
 
   test('should call estimateGasForErc20 with token', () => {
-    expect(estimateGas(mock.tx, mock.erc20, mock.api)).resolves.toEqual(
-      new BigNumber(153.75)
+    expect(estimateGas(mock.txEth, mock.erc20, mock.api)).resolves.toEqual(
```

Comment:
i think this should be `mock.txErc20` instead of `mock.txEth`

suggest defining `txEth` and `txErc20` in testHelpers/mock.js as follows:
```
export const txEth = {
  amount: AMOUNT, // In Ether
  from: ADDRESS_FROM,
  gasPrice: GAS_PRICE,
  to: ADDRESS_TO,
  token: eth
};

const txErc20Base = {
  amount: AMOUNT, // In token
  from: ADDRESS_FROM,
  gasPrice: GAS_PRICE,
  to: ADDRESS_TO,
  token: erc20
};

export const txErc20 = {
  ...txErc20Base,
  args: [
    txErc20Base.to,
    new BigNumber(txErc20Base.amount).multipliedBy(
      new BigNumber(10).pow(txErc20Base.token.decimals)
    )
  ],
  options: {
    from: txErc20Base.from,
    gasPrice: toWei(txErc20Base.gasPrice, 'shannon') // shannon == gwei
  }
};
```